lib/nebulex_redis_adapter.ex

```diff
@@ -368,17 +373,28 @@ defmodule NebulexRedisAdapter do
         children: children
       )
 
+    stats_counter = Stats.init(opts)
+
     meta = %{
       name: name,
       mode: mode,
       keyslot: keyslot,
       nodes: nodes,
       pool_size: pool_size,
+      stats_counter: stats_counter,
+      started_at: DateTime.utc_now(),
       default_dt: Keyword.get(opts, :default_data_type, :object),
       telemetry: Keyword.fetch!(opts, :telemetry),
       telemetry_prefix: Keyword.fetch!(opts, :telemetry_prefix)
     }
 
+    Telemetry.attach_many(
```

Comment:
Let's do this to avoid dialyzer complaining:

```
:ok =
  Telemetry.attach_many(
    stats_counter,
    [meta.telemetry_prefix ++ [:command, :stop]],
    &StatsHandler.handle_event/4,
    stats_counter
  )
```